### DISCUSSION TITLE
Jupyter Notebook Error - The solver StabilityDerivatives cannot be found in the list of solvers.

### DISCUSSION THREAD

**Discussion by jas220** [ImperialCollegeLondon/sharpy#155] [Q&A] [answered]
Hello again,

I have successfully been able to install and run SHAPy installation tests on a VM. I've been able to run the Flutter Analysis of a Goland Wing using the SHARPy Linear Solver example and the T-Tail HALE Model tutorial. Now I am trying to run the Asymptotic Stability of a Flying Wing in Cruise Trimmed Conditions Example, but I am getting this error. I have loaded the sharpy environment variables and activated the `sharpy_env` on the Ananconda Navigator before running Jupyter Notebook. I have also run it directly from the SHARPy directory `sharpy/docs/source/content/example_notebooks/` without having to download it. This is the error I get.
<img width="583" alt="Screen Shot 2021-05-13 at 4 45 07 PM" src="https://user-images.githubusercontent.com/82110485/118154648-37496c80-b40f-11eb-9e7c-d8e82a01533c.png">

PS: I would still like to get help on how to solve [this](https://github.com/ImperialCollegeLondon/sharpy/discussions/147) issue because running this VM on my mac is very slow, especially when I want to open the files in ParaView; I just wanted to get started on using SHARPy to not fall behind on my project.

Thank you so much in advance!

**Comment by ngoiz** [comment, marked as answer]
Hi, yes, the `StabilityDerivatives` post-proc got deprecated and was removed (I forgot to update the test case). Just remove it from the `flow` and it should work. Rest assured we are thinking about possible solutions to the other issue
